parcels/application_kernels/advectiondiffusion.py

```diff
@@ -1,6 +1,6 @@
 """Collection of pre-built advection-diffusion kernels.
 
-See `this tutorial <https://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_diffusion.ipynb>`__ for a detailed explanation.
+See `this tutorial <../examples/tutorial_diffusion.ipynb>`__ for a detailed explanation.
```

Comment:
Would this link work, given this file is not in the docs? Better to change to https://docs.oceanparcels.org/en/latest/examples/tutorial_diffusion.html (I think that'll be the correct link)?